multiregion/docs/multiregion.rst

```diff
@@ -447,7 +534,92 @@ There is a script you can run to collect the JMX metrics from the command line,
       multi-region-sync: 4
       multi-region-async: 4
       multi-region-default: 4
+      multi-region-async-op-under-min-isr: 4
+      multi-region-async-op-under-replicated: 4
+      multi-region-async-op-leader-is-observer: 4
+
+      ==> Monitor ObserversInIsrCount
+
+      single-region: 0
+      multi-region-sync: 0
+      multi-region-async: 0
+      multi-region-default: 0
+      multi-region-async-op-under-min-isr: 0
+      multi-region-async-op-under-replicated: 0
```

Comment:
QQ: Should this have a non zero value if we've promoted observers in this example?

Reply:
yes, but these metrics are shown before region failure and hence no observers have been promoted.

Reply:
@skaundinya15 I'm reworking the docs with more JMX meric output, so this will become clearer based on when the JMX metrics are captured.  Please stay tuned...PR in progress ;)

Reply:
Makes sense, thanks @thomaskwscott. Looking forward to that @ybyzek 💯